cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -411,16 +415,22 @@ func transcribeFiles(workerID int, wg *sync.WaitGroup, client *cubic.Client,
 			audioChannelsUint32 = append(audioChannelsUint32, uint32(c))
 		}
 
+		cfg := &cubicpb.RecognitionConfig{
+			ModelId:       model,
+			AudioEncoding: audioEncoding,
+			IdleTimeout:   &pbduration.Duration{Seconds: 30},
+			AudioChannels: audioChannelsUint32,
+		}
+
+		if outputFormat == "timeline" {
+			cfg.EnableWordConfidence = true
+			cfg.EnableWordTimeOffsets = true
+			cfg.EnableRawTranscript = true
```

Comment:
Once we've updated cubicsvr so it can output formatted transcript *and* word-level features, remove this line

Reply:
Should that be added as a `//TODO(julie):` line in the code?  I'm going to forget about it if it's just here in a PR comment.